.github/workflows/llvmdev_win_builder.yml

```diff
@@ -0,0 +1,51 @@
+name: llvmdev_win_builder
```

Comment:
This is a adaptation of https://github.com/numba/llvmlite/pull/1106/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR162-R207

This workflow is saved here for record keeping purpose. No automatic trigger.

This workflow was used to produce the llvmdev_manylinux build on https://anaconda.org/numba/llvmdev/15.0.7/download/win-64/llvmdev-15.0.7-manylinux.tar.bz2